xlive/H2MOD/Variants/Infection/Infection.cpp

```diff
@@ -222,6 +225,41 @@ void Infection::setPlayerAsZombie(int playerIndex) {
 	call_give_player_weapon(playerIndex, e_weapons_datum_index::energy_blade, 1);
 }
 
+void Infection::onGameTick()
+{
+	if(Engine::get_game_life_cycle() == _life_cycle_in_game && NetworkSession::LocalPeerIsSessionHost())
+	{
+		PlayerIterator playerIt;
+		int human_count = 0;
+		int zombie_count = 0;
+		while (playerIt.get_next_active_player())
+		{
+			XUID playerIdentifier = playerIt.get_current_player_id();
+			bool isZombie = std::find(Infection::zombieIdentifiers.begin(), Infection::zombieIdentifiers.end(), playerIdentifier) != Infection::zombieIdentifiers.end();
+
+			if (isZombie)
+				zombie_count++;
+			else
+				human_count++;
+
+		}
+		if(human_count == 0 && zombie_count >= 1)
+		{
+			ticks_without_any_humans++;
+		}
+		else
+		{
+			ticks_without_any_humans = 0;
+		}
```

Comment:
This can use a ternary operator
`(human_count == 0 && zombie_count >= 1 ? ticks_without_any_humans++ : ticks_without_any_humans++);`

Reply:
no